src/styles/components/_datepicker.scss

```diff
@@ -0,0 +1,140 @@
+@use "sass:math";
+@use "sass:color";
+@use "styles/abstracts" as *;
+
+// Note: Not sure this is the right way to import...
+@import "@meemoo/react-components/node_modules/react-datepicker/dist/react-datepicker.min.css";
```

Comment:
We hebben hier 2 opties:
1. We re-exporten de file vanuit `@meemoo/react-components/css/vendor/react-datepicker.min.css`
2. We voegen react-datepicker toe als peerDep in `@meemoo/react-components` en dan kan je hier dus gwn vanuit die package importen
Ik denk dat optie 1 het makkelijkste gaat zijn dan moet je geen extra deps in de client installeren.

Reply:
@andrych Ben niet mee met wat je hier van plan bent. Gaan we de file dupliceren uit `react-datepicker`, in de `components`-repo onder `/css/vendor/react-datepicker.min.css` plaatsen en dan importeren hier in de `client`-repo?

Eigenlijk zou ik verwachten dat we deze import volledig kunnen weglaten door https://github.com/viaacode/react-components/blob/master/src/components/Datepicker/Datepicker.scss#L1 maar dat is niet het geval, is dat een rollup config die we kunnen aanpassen?